<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="pc-PC" ignored="0" name="test" tests="1" failures="0" timestamp="22 Aug 2018 08:50:36 GMT" time="27.664" errors="0">
  <testcase name="@AfterMethod tearDown" time="2.135" classname="com.flipkart.qa.testcases.LoginTestEA">
    <failure type="org.apache.commons.mail.EmailException" message="Sending the email to the following server failed : smtp.gmail.com:465">
      <![CDATA[org.apache.commons.mail.EmailException: Sending the email to the following server failed : smtp.gmail.com:465
at org.apache.commons.mail.Email.sendMimeMessage(Email.java:1242)
at org.apache.commons.mail.Email.send(Email.java:1267)
at com.flipkart.qa.util.SendMail.sendEMail(SendMail.java:22)
at com.flipkart.qa.base.TestBase.getresult(TestBase.java:116)
at com.flipkart.qa.testcases.LoginTestEA.tearDown(LoginTestEA.java:75)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: javax.mail.AuthenticationFailedException
at javax.mail.Service.connect(Service.java:319)
at javax.mail.Service.connect(Service.java:169)
at javax.mail.Service.connect(Service.java:118)
at javax.mail.Transport.send0(Transport.java:188)
at javax.mail.Transport.send(Transport.java:118)
at org.apache.commons.mail.Email.sendMimeMessage(Email.java:1232)
... 27 more
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="loginfunctionality" time="13.854" classname="com.flipkart.qa.testcases.LoginTestEA"/>
</testsuite> <!-- test -->
